scanners/ionic/ionic.go

```diff
@@ -2,6 +2,7 @@ package ionic
 
 import (
 	"fmt"
+	"github.com/bitrise-io/bitrise-init/scanners/nodejs"
```

Comment:
Usually we organise the imports into groups. The first group is for the stdlib imports and then comes the rest. You can set up Goland to automatically use `goimports` when you save a file.